apps/mobile/src/icons/QuestionCircleIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { useColorScheme } from 'nativewind';
+import Svg, { Circle, Path, Rect } from 'react-native-svg';
+
+import colors from '~/shared/theme/colors';
+type Props = {
+  width?: number;
+};
+
+export function QuestionCircleIcon({ width = 24 }: Props) {
+  const { colorScheme } = useColorScheme();
+  return (
+    <Svg width={width} height={width} viewBox="0 0 24 25" fill="none">
+      <Rect
+        x="-0.5"
+        y="0.5"
+        width={width - 1}
+        height={width - 1}
```

Comment:
Just wondering why do we minus one here? 🤔

Reply:
that was how figma exported it , I think basically the inside content is supposed to be slightly smaller than the parent wrapper